tests/Fixer/Phpdoc/NoSuperfluousPhpdocTagsFixerTest.php

```diff
@@ -102,6 +104,18 @@ class Foo {
      */
     public function doFoo($bar) {}
 }',
+                ['allow_mixed' => false],
+            ],
+            'allow_mixed=>true' => [
+                '<?php
+class Foo {
+    /**
+     *
```

Comment:
The `@param mixed $bar` annotation is missing here (to prove the fixer doesn't remove it). Maybe a `@return mixed` would be nice too.